engine/gamesys/src/gamesys/resources/res_material.cpp

```diff
@@ -158,7 +208,7 @@ namespace dmGameSystem
                 (dmVMath::Vector4*) vertex_constant[i].m_Value.m_Data, vertex_constant[i].m_Value.m_Count);
         }
 
-
+        // Deprecated
```

Comment:
This block is 8 years old and the "textures" field in the material proto is 14 years old.. maybe it's time we just remove it from the engine completely.. if we don't want to remove the field in the DDF, maybe we could just notify them it's no longer supported